src/wasm-lib/justfile

```diff
@@ -4,7 +4,7 @@ new-test name:
     TWENTY_TWENTY=overwrite cargo nextest run --test executor -E 'test(=visuals::{{name}})'
 
 lint:
-    cargo clippy --all --tests --benches -- -D warnings
+    cargo clippy --all --all-targets --tests --benches -- -D warnings
```

Comment:
If --all-targets includes tests and benches, then we should remove --tests and --benches right? They're unnecessary and implied by --all-targets?

Reply:
Done.